course-project/Backend/src/cmd/rest-api/main.go

```diff
@@ -13,12 +13,11 @@ import (
 func dbsFromEnv() db.DBSetup {
 	dbs := db.DBSetup{
 		User:   utils.EnvOrDef("POSTGRES_USER", "postgres"),
-		Passwd: utils.EnvOrDef("POSTGRES_PASSWORD", "postgres"),
-		Host:   utils.EnvOrDef("DB_HOST", "postgres"),
-		//postgres://postgres:postgres@postgres:5432/postgres
-		Port: utils.EnvOrDefInt("DB_PORT", 5432),
-		Name: utils.EnvOrDef("POSTGRES_DB", "postgres"),
-		Type: "postgres",
+		Passwd: utils.EnvOrDef("POSTGRES_PASSWORD", "1234"),
```

Comment:
After these changes, server can no longer connect to database container

Reply:
fixed it by adding appropriate env variables to docker-compose

Reply:
It still fails to connect. You have localhost as host, which is wrong, because in docker-compose you have postgres